### ISSUE TITLE
We're installing `terryfy` with these wheels

### ISSUE THREAD

**Issue by jamadden** [zopefoundation/BTrees#98]
Just like with [persistent](https://github.com/zopefoundation/persistent/issues/99). The fix is [probably the same](https://github.com/zopefoundation/persistent/pull/100)

```console
$ wget https://files.pythonhosted.org/packages/10/71/3f6de6470513f11311bc91aeae374c0cc539cf237656853c8bef08001dd9/BTrees-4.5.1-cp27-cp27m-macosx_10_6_intel.whl
...
2018-10-22 16:33:02 (2.79 MB/s) - ‘BTrees-4.5.1-cp27-cp27m-macosx_10_6_intel.whl’ saved [838249/838249]

$ unzip -l BTrees-4.5.1-cp27-cp27m-macosx_10_6_intel.whl
...
     2703  08-10-2018 11:05   BTrees/tests/test_utils.py
       31  08-10-2018 11:05   terryfy/__init__.py
      811  08-10-2018 11:05   terryfy/bdist_wheel.py
      720  08-10-2018 11:05   terryfy/cp_suff_real_libs.py
     1099  08-10-2018 11:05   terryfy/fuse_suff_real_libs.py
     1708  08-10-2018 11:05   terryfy/monkeyexec.py
     1018  08-10-2018 11:05   terryfy/repath_lib_names.py
     2148  08-10-2018 11:05   terryfy/test_travisparse.py
     1763  08-10-2018 11:05   terryfy/travisparse.py
     9118  08-10-2018 11:05   terryfy/wafutils.py
```

**Comment by azmeuk** [comment]
Ain't this ticket is solved by db1c8b9c986d3f764c84db4e9efc1239d1d0ae95?

**Comment by jamadden** [comment]
Yes, #99 solved this. Thanks.
